### PR TITLE
Made the project even more awesome 😎

### PR DESCRIPTION
**Pull request by Zedfrigg**
Hey fellow code artisans, I just made a couple of changes that you'll 💚. Now it's 200% more webscale 🌐🌐🌐 and uses AI 🤖🤖🤖 and machine learning! Future here we come!!!11

### CODE REVIEW

**Comment by Zedfrigg** [comment]
PS i don't really know what webscale is but it kinda sounds like limescale and i heard Google likes it which is good cuz i wanna work for them #hireme 🤪🤪🤪🤪🤪

**Comment by Zedfrigg** [comment]
Can we please ban Travis? He's calling @Zedfrigg's build a failure which is a kind of attitude we don't need in this community. We're all about positivity and diversity, and we want this repo to be a safe space in which people can express themselves. Unless the maintainers don't like that expression of course.

- Katryn Sojawski

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "andyz",
-  "version": "1.0.6",
+  "version": "1.0.6-PRERELEASE-SNEAKPREVIEW-COMINGSOONTOACINEMANEARYOU",
```
Comment (ricardovogel) [inline package.json]: Changes look good in general. But Andy lectures are a Collegerama™ Original, and therefore will not be aired in cinema. Please change this to "1.0.6-PRERELEASE-SNEAKPREVIEW-COMINGSOONTOACOLLEGERAMANEARYOU".

Comment (Zedfrigg) [reply package.json]: Understandable, my attorney agrees.

**Pull request merged**